src/IdentityServer/Constants.cs

```diff
@@ -115,7 +115,7 @@ public static class SigningAlgorithms
         //OidcConstants.PromptModes.Create, 
     };
 
-    public const string SuppressedPrompt = "suppressed_" + OidcConstants.AuthorizeRequest.Prompt;
+    public const string ProcessedPrompt = "processed_" + OidcConstants.AuthorizeRequest.Prompt;
```

Comment:
Renaming here makes the names consistent, but is a (very minor) breaking change.

Reply:
This would only break someone in-flight, right?

Reply:
Yeah, if you're using the constant or looking at the parameters raw and using the old name, this means you need to make an update.

Reply:
Leave the old constant and deprecate it w/ obsolete, please.